flutter-gui-tests/testSrc/io/flutter/tests/gui/PerfTest.kt

```diff
@@ -28,24 +31,10 @@ class PerfTest : GuiTestCase() {
       launchFlutterApp()
       val inspector = flutterPerfFixture(this)
       inspector.populate()
-    }
-  }
 
-  // TODO Share with InspectorTest
-  fun IdeFrameFixture.launchFlutterApp() {
-    step("Launch Flutter app") {
-      findRunApplicationButton().click()
-      val runner = runner()
-      Pause.pause(object : Condition("Start app") {
-        override fun test(): Boolean {
-          return runner.isExecutionInProgress
-        }
-      }, Timeouts.seconds30)
-    }
-  }
+      pause()
 
-  // TODO Share with InspectorTest
-  private fun IdeFrameFixture.runner(): ExecutionToolWindowFixture.ContentFixture {
-    return runToolWindow.findContent("main.dart")
+      runner().stop()
+    }
   }
 }
```

Comment:
nit: have you run the kotlin formatter on these files? I would expect it would fix the missing trialing newline, etc.

Reply:
Yes, I did